lib/rspotify/user.rb

```diff
@@ -394,8 +394,8 @@ def remove_albums!(albums)
     def save_albums!(albums)
       albums_ids = albums.map(&:id)
       url = "me/albums"
-      request_body = albums_ids.inspect
-      User.oauth_put(@id, url, request_body)
+      request_body = { ids: tracks_ids }
```

Comment:
@dildal shouldn't it be `albums_ids` instead of `tracks_ids`?

Reply:
sorry, sloppy copy paste job 🤦